salt/etcd/create-or-update-etcd-pillar

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker exec $(docker ps -a | grep velum-dashboard | awk '{print $1}') entrypoint.sh bundle exec rails runner 'ActiveRecord::Base.logger=nil; pillar = Pillar.find_or_create_by(pillar: "api:etcd_version"); pillar.value = "etcd3"; pillar.save; Velum::Salt.orchestrate'
```

Comment:
Do we need the `Velum::Salt.orchestrate` at the end? We are doing a `highstate: True` as a last step in `orch/etcd-migrate.sls`...

Reply:
Agree, as running two orchestrations at once will fail randomly (we've seen this time+time again with the etc-hosts orchestration, we simply can't run two orchestrations at once)